client/my-sites/plans/current-plan/current-plan-thank-you-card/paid-plan-thank-you-card.js

```diff
@@ -22,6 +23,15 @@ export class PaidPlanThankYouCard extends Component {
 		return `/plans/my-plan/${ siteSlug }`;
 	}
 
+	componentDidUpdate() {
+		if ( this.props.progressComplete === 100 ) {
```

Comment:
This has a possibility of recording the tracks event any number of times. `componentDidUpdate` may be called any number of times, we don't want to enter the block on every invocation, but only when `was < 100 && is == 100`:

```suggestion
		if ( prevProps.progressComplete < 100 && this.props.progressComplete >= 100 ) {
```

There's an example of just that in the docs because it's such a frequent pitfall: https://reactjs.org/docs/react-component.html#componentdidupdate